content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/autocomponents.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Multiplayer Autocomponents
+title: Multiplayer AutoComponents
 description: A reference for defining Open 3D Engine multiplayer state through autocomponents.
-linktitle: Autocomponents
+linktitle: AutoComponents
 ---
 
-*Autocomponents* provide a convenient way to define states of a multiplayer component relevant to network synchronization. Using the [AzAutoGen](/docs/user-guide/engine/autogen) system, autocomponent files found inside of your project are processed during builds to create C++ classes for components and controllers that provide network replication and remote function calls. Autocomponents also take care of Editor and behavior context bindings so that the bound component shows in the Editor and works with O3DE scripting.
+*AutoComponents* provide a convenient way to define states of a multiplayer component relevant to network synchronization. Using the [AzAutoGen](/docs/user-guide/engine/autogen) system, autocomponent files found inside of your project are processed during builds to create C++ classes for components and controllers that provide network replication and remote function calls. AutoComponents also take care of Editor and behavior context bindings so that the bound component shows in the Editor and works with O3DE scripting.
```

Comment:
I think we should spell autocomponent as auto-component.  pinging @sptramer

Reply:
I went with our in code naming but I'm open here. I just felt like Autocomponents and Autopackets didn't necessarily read well.